### ISSUE TITLE
CONTROL_SELECTOR_STRING controls show localized string reference

### ISSUE THREAD

**Issue by Florob** [christophe-calmejane/Hive#166]
Controls of value type `CONTROL_SELECTOR_STRING` show the localized string references instead of the actual strings they refer to. I doubt that is the intention.

The attached AEM contains one as control 38 (last one in the `AUDIO_UNIT`).

[Entity_0x480BB2FFFED00013.ave.gz](https://github.com/christophe-calmejane/Hive/files/13949921/Entity_0x480BB2FFFED00013.ave.gz)

**Comment by christophe-calmejane** [comment]
Thanks Florian, I'll take a look. Have a few other issues to fix before next release, but I'll check this one as well.

**Comment by christophe-calmejane** [comment]
It now looks like this, is that correct?
![Image](https://github.com/user-attachments/assets/b5e4707d-c7c7-4463-804e-7b2c5b9bd6ae)

![Image](https://github.com/user-attachments/assets/2b7ee121-c3fd-4a39-9d41-cb3a8e70fa75)

**Comment by Florob** [comment]
No, these are still the string references. The options should be:
* Auto
* Short Delay Sharp
* Short Delay Slow
* Sharp
* Slow

**Comment by christophe-calmejane** [comment]
Ok got it, it was not even unpacking with the previous Hive version so I wasn't sure.
